todo_app/tests/conftest.py

```diff
@@ -0,0 +1,73 @@
+import os
+from time import sleep
+from threading import Thread
+
+import pytest
+from dotenv import find_dotenv, load_dotenv
+from selenium import webdriver
+
+from todo_app.data.trello_items import create_board, delete_board
+from todo_app.data.view_model import ViewModel
+from todo_app.data.item import Item
+from todo_app import app
+
+TIME_IN_SECONDS = 1
+
+
+@pytest.fixture
+def load_environment_variables():
+    # Use our test integration config instead of the 'real' version
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+
+
+@pytest.fixture
+def client(load_environment_variables):
+    # Create the new app.
+    test_app = app.create_app()
+    # Use the app to create a test_client that can be used in our tests.
+    with test_app.test_client() as client:
+        yield client
+
+
+@pytest.fixture
+def example_view_model_items(load_environment_variables):
+    # Create mock items
+    item1 = Item(title="Task 1", id_list=os.getenv("TRELLO_TODO_LIST_ID"))
+    item2 = Item(title="Task 2", id_list=os.getenv("TRELLO_DOING_LIST_ID"))
+    item3 = Item(title="Task 3", id_list=os.getenv("TRELLO_DONE_LIST_ID"))
+    item4 = Item(title="Task 4", id_list=os.getenv("TRELLO_TODO_LIST_ID"))
+    item5 = Item(title="Task 5", id_list=os.getenv("TRELLO_DOING_LIST_ID"))
+    item6 = Item(title="Task 6", id_list=os.getenv("TRELLO_DONE_LIST_ID"))
+
+    return ViewModel([item1, item2, item3, item4, item5, item6])
+
+
+@pytest.fixture(scope='module')
+def app_with_temp_board():
+    # Create the new board & update the board id environment variable
+    temp_board = create_board("Temp Board For Testing")
+    os.environ['TRELLO_BOARD_ID'] = temp_board["id"]
```

Comment:
This fails for me because it is still using the fake environment variables by default, which means it fails to create the new board - if we add a line in to load the real environment variables for this test that can fix this (as below) although we should be a bit wary that this won't spill over into other tests!
```suggestion
def app_with_temp_board():
    load_dotenv(override=True)
    # Create the new board & update the board id environment variable
    temp_board = create_board("Temp Board For Testing")
    os.environ['TRELLO_BOARD_ID'] = temp_board["id"]
```

One way to do this is to move the end-to-end tests into their own folder, and then we can explicitly run them separately when we want to